config/parameters/fc/base.txt

```diff
@@ -5,3 +5,7 @@ fc.leader.fire_time_near   300
 
 fc.follower.fire_time_far     1800
 fc.follower.fire_time_near    300
+
+# Ideally these would be in an actuators folder...
+fc.leader.thruster.noise_radius 1.0
```

Comment:
renaming this thruster.noise.1sig would be more accurate just so we know it is 1sig

Reply:
will rename!

Reply:
i think _sigma is sufficient